packages/components/atoms/f-filter-pill/src/components/FilterPill.vue

```diff
@@ -154,17 +158,29 @@ $filter-pill-checkbox-width: 16px;
     align-self: center;
     opacity: 0;
     width: 0;
+    min-width: 0;
+    transform: translate3d(0, 0, 0) scale(0) rotate(-45deg);
+    will-change: transform, opacity, width, min-width, margin-right;
```

Comment:
This `will-change` property might not be necessary. It sounds as though it's only worth adding as a last resort for known performance issues, however I'm not an expert!

[Reference](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change)

Reply:
Good point! I removed it for now. Let's see if we have any performance along the way.